Ashe-Oro/bark-power-tools
# feat: Add Barks Back Display to Account Details

# Add Barks Back Display Feature

## Description
This PR adds support for displaying the "Barks Back" metric, which is fetched from the barking-power endpoint. The value is now displayed alongside other basic account details in both basic and detailed views.

## Changes
- Added new `fetchBarksBack` method to `BarkApi` class
- Updated display methods in `BarkView` to include Barks Back value
- Modified data processing methods in `BarkManager` to fetch and handle Barks Back data
- Added console logging for debugging raw Barks Back values
- Implemented error handling and default values to prevent undefined displays

## API Integration
Integrates with endpoint: https://sure-angeline-piotrswierzy-b061c303.koyeb.app/barking-power/

## Testing
1. Search for an account using either Twitter handle or account ID
2. Verify Barks Back value appears in the basic details section
3. Check console for raw value logging
4. Verify no undefined values appear in the UI
5. Confirm formatting matches existing numerical displays

## Screenshots
[Add screenshots of the new feature here]

## Type of Change

## Checklist
- [x] Code follows existing style guidelines
- [x] Added console logging for debugging
- [x] Tested with multiple accounts
- [x] Updated all relevant display methods
